Sources/KituraContracts/CodableQuery/Coder.swift

```diff
@@ -16,21 +16,44 @@
 
 import Foundation
 
-/// Class defining shared resources for the QueryDecoder and QueryEncoder
+/**
+ Class defining shared resources for the QueryDecoder and QueryEncoder.
+ 
+ ### Usage Example: ###
+ ````swift
+ let date = Coder().dateFormatter.date(from: "2017-10-31T16:15:56+0000")!
```

Comment:
@ianpartridge - why would we need to have the force unwrap operator at the end of the String format which we’re passing in when dateFormatter.date() returns !String

I’m looking for clarification because confusingly this code compiles within Xcode both with and without the ! at the end, so I need someone with a larger brain than mine to explain why the sample code needs a ! on it.